Makefile.eve

```diff
@@ -15,6 +16,13 @@ ifeq ($(BUILDKIT_PROGRESS),)
     export BUILDKIT_PROGRESS := plain
 endif
 
+# if BRANCH is defined form a command line we are running under CI
+# fin not theen add user name to DIRTY to avoid docker tag conflicts
```

Comment:
fin? s/theen/then/

Reply:
<facepalm> idk what happened to my spelling

Reply:
fixed